public/animatemap-onboard/step-07.svg

```diff
@@ -0,0 +1,65 @@
+<svg width="404" height="230" viewBox="0 0 404 230" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Inside assets: 2
Outside assets:  60+

🤷 At this point renaming `assets`->`banner` might be more in line with the established standard

![sparkle-assets-01](https://user-images.githubusercontent.com/79229621/130443507-ca7c8618-2e7f-4afd-ace7-c39b1961c877.png)
![sparkle-assets-02](https://user-images.githubusercontent.com/79229621/130443517-464c2651-1bf4-4a05-ae52-9a1b322a72b2.png)

Reply:
@goran-peoski-work `src/assets` that's where we've been adding assets in the past month or two, they are loaded with the app and are not flickering and fading in when they are loaded on the page. Where the public images do that sometimes. Maybe because they are not lazy loaded.

Reply:
Thanks for the tip. No flicker anymore 3f91915d8790fc5382736f905021ca64c1b06028